vendor/golang.org/x/sys/unix/ztypes_linux_amd64.go

```diff
@@ -694,3 +694,10 @@ type Termios struct {
 	Ispeed uint32
 	Ospeed uint32
 }
+
+type Winsize struct {
```

Comment:
__[golint]__
* [golint] exported type Winsize should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->